From fd36aee4559102457d2ad6f86d5611c76b4072e7 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Tue, 1 Aug 2023 10:41:39 +0200
Subject: [PATCH] configure.ac: don't use -L/usr/local/lib in LDFLAGS

* fixes:
  ERROR: presage-0.9.1-r0 do_configure: QA Issue: This autoconf log indicates errors, it looked at host include and/or library paths while determining system capabilities.
  Rerun configure task after fixing this. [configure-unsafe]

  presage/0.9.1-r0/git $ grep unsafe ../build/config.log
  /OE/build/luneos-nanbield/tmp-glibc/work/corei7-64-webos-linux/presage/0.9.1-r0/recipe-sysroot-native/usr/bin/x86_64-webos-linux/../../libexec/x86_64-webos-linux/gcc/x86_64-webos-linux/13.2.0/ld: warning: library search path "/usr/local/lib" is unsafe for cross-compilation
  ...

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
Upstream-Status: Pending
---
 configure.ac | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index 48747e8..c933e19 100644
--- a/configure.ac
+++ b/configure.ac
@@ -89,9 +89,6 @@ AM_CONDITIONAL([HAVE_LD_WITH_VERSION_SCRIPT],
                [test "x$have_ld_with_version_script" = "xyes"])
 
 
-dnl Cygwin windows specific flag
-LDFLAGS="-L/usr/local/lib $LDFLAGS"
-
 dnl ====================
 dnl Checks for libraries
 dnl ====================
